pkg/detectors/packagecloud/packagecloud.go

```diff
@@ -0,0 +1,73 @@
+package packagecloud
+
+import (
+	"context"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"packagecloud"}) + `\b([0-9a-z]{48})\b`)
```

Comment:
suggestion: this regex can be made a bit stricter `\b([0-9a-f]{48})\b` You can only look for `a-z` since this is an encoded.

Reply:
Thanks! I just pushed the enhancement.

Reply:
Some extra context: it's hex encoding that uses the a-f characters rather than a-z